03_gamedesign/index.tex

```diff
@@ -1,5 +1,75 @@
+\end{definition}
+
+\begin{definition} \label{def:turn}
+    A \textbf{turn} is defined as a series of exchanges between agents in which islands receive resource updates, attend Inter-Island Organisation meetings, and interact with one another and the game state through \textbf{actions}. A turn can be broken down as follows:
+    \begin{enumerate}
+        \item Resource Updates for each island and on global game state.
+        \item IIGO decides rule changes, elections, sanctions.
+        \item IIFO provides a forum for information exchange to mitigate both short and long term risk dilemmas.
+        \item IITO facilitates gift exchanges and allows agents to communicate to make deals between one another without organisation supervision.
+        \item Islands submit decisions on their actions to the server to formally end the turn.
+        \item Check if a disaster occurs this turn.
+        \item Server processes actions and updates game and island states.
+            \begin{itemize}
+                \item A cost of living is subtracted from an islands pool before the next term. This is the simulation-level equivalent to using resources to stay alive (e.g. food consumed). These resources are permanently consumed and do NOT go into the common pool. Note: this is NOT the same as the tax.
+                \item Check if the game is over.
+                \item Check if any islands are \textbf{critical} (i.e. below the threshold).
+                \item Check if any islands are \textbf{dead}.
+            \end{itemize}
+    \end{enumerate}        
+\end{definition}
+
+\begin{definition} \label{def:gameseason}
+    A \textbf{season} is defined as a series of turns and concludes with a disaster. Seasons formalise the flow of the game and provide a method to track the number of disasters the islands survive.
+\end{definition}
+
+\begin{definition} \label{def:gamestate}
+    The \textbf{game state} is the set of information an island receives at the start of each turn. This includes, but is not limited to, the resources it was allocated the previous turn, the geographical location of the islands, the quantity of resources left in the common pool and the set of rules for the turn (including any rules that were modified the previous turn). It is important to note that each island's game state is private, so no other island can see another island's resource allocation.
+\end{definition}
+
+\begin{definition} \label{def:gameaction}
+    An \textbf{action} is a decision an island can make or that is made by any Inter-Island Organisation that updates the game state. Actions can be made at different stages within a turn both within Inter-Island Organisations and at the end of a turn. This includes but is not limited to taking resources from or donating resources to from the common pool, rule changes in the IIGO, and gift requests or acceptances. 

```

Comment:
```suggestion
    An \textbf{action} is a decision an island can make or that is made by any inter-island organisation that updates the game state. Actions can be made at different stages within a turn both within Inter-Island Organisations and at the end of a turn. This includes but is not limited to taking resources from or donating resources to from the common pool, rule changes in the IIGO, and gift requests or acceptances. 

```

can you given example for this ''inter-island organisation''? 
do you mean IITO/IIFO/IIGO or all of them?

Reply:
all of them - also Inter Island Organisation refers to the organisational bodies acronymed by IIGO, IITO, IIFO so Inter Island Organisation should be capital here